cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -2021,6 +2021,12 @@ public static final class Tethering {
     public static final String CONNECTION_TIMEOUT_SECONDS = "tethering.connection.timeout.seconds";
     public static final int DEFAULT_CONNECTION_TIMEOUT_SECONDS = 60;
 
+    /**
+     * Tethering client timeouts.
+     */
+    public static final String CLIENT_CONNECTION_TIMEOUT_MILLIS = "tethering.client.connection.timeout.millis";
```

Comment:
how is this different than the connection timeout above? Would be good to add that to the comments